### ISSUE TITLE
Bug: "Some values are not from the provided attribute."

### ISSUE THREAD

**Issue by yonathan9669** [saleor/saleor#15608]
### What are you trying to achieve?

I'm building a Saleor APP to integrate a custom ERP.

I have successfully imported:
- ProductTypes
- Attributes
- Categories
- ...

I also have already completed the Products & Variants import process. Yet today after upgrading my local development stack from `3.19.4` to `3.19.10` I suddenly start hitting the error:

> `Some values are not from the provided attribute.`

Which was really weird because it was fully working just before the upgrade.

> Successfully imported product/variant before upgrade
![imagen](https://github.com/saleor/saleor/assets/897389/ec6e5098-335d-4459-80f3-a3d41e7f7871)

I literally spent ~3h debugging my code base to end up finding it was a problem with the `saleor` upgrade. I definitively knew it when I did the downgrade and everything was fully operational again.

### Steps to reproduce the problem

I'm not quite sure what brings the problem up since some products (those with few choices) were being properly imported.

Anyway I could say that the variant with a total of `49` choices across `10` selectable attributes, was the one throwing this Error.
> Count of choices per attribute.
`[    1,    1,    1,    1,    2,    1,    18,    20,    1,    3]`

As the Error is quite generic I can't know which is the attribute/choice(s) that are bringing this error.

---
I think you could reproduce it by inserting this ProductType and all the Attributes. One of the `inputs` used that triggers this exception is provided as part of the logs coupled with the error.

> ProductType
```js
{
  id: "UHJvZHVjdFR5cGU6NA==",
  hasVariants: true,
  productAttributes: [],
  assignedVariantAttributes: [
    {
      attribute: {
        id: "QXR0cmlidXRlOjEz",
        slug: "transmission",
        inputType: "DROPDOWN",
        valueRequired: true,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTYw",
                slug: "mecanica",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTY0",
                slug: "automatica",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTY2",
                slug: "cvt",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTY5",
                slug: "automatizada",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjE0",
        slug: "bodywork",
        inputType: "DROPDOWN",
        valueRequired: true,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTYx",
                slug: "sedan",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTYy",
                slug: "hatchback",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTY1",
                slug: "suv",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTY4",
                slug: "coupe",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTcw",
                slug: "cabina-simple",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTcy",
                slug: "station-wagon",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTc0",
                slug: "otro",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTc2",
                slug: "cabina-y-media",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTc3",
                slug: "comercial-furgon",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTc5",
                slug: "minibus",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTgy",
                slug: "van",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTg1",
                slug: "cabriolet-descapotable",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTg4",
                slug: "camioneta",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTky",
                slug: "motor-home",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjE5",
        slug: "number_velocity",
        inputType: "DROPDOWN",
        valueRequired: false,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjA0",
                slug: "cvt",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjA2",
                slug: "4",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjA4",
                slug: "5",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjEw",
                slug: "6",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjEx",
                slug: "7",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjEz",
                slug: "8",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjE0",
                slug: "9",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjE1",
                slug: "10",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjE3",
                slug: "0",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjE5",
                slug: "3",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjIx",
                slug: "11",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjE4",
        slug: "cylender",
        inputType: "PLAIN_TEXT",
        valueRequired: false,
        choices: {
          edges: [],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjE1",
        slug: "fuel",
        inputType: "DROPDOWN",
        valueRequired: true,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTYz",
                slug: "bencina",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTY3",
                slug: "diesel",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTcx",
                slug: "hibrido",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTcz",
                slug: "electrico",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTc1",
                slug: "gas",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjIw",
        slug: "power",
        inputType: "PLAIN_TEXT",
        valueRequired: false,
        choices: {
          edges: [],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjIx",
        slug: "aesthetic_equipment",
        inputType: "MULTISELECT",
        valueRequired: false,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjI2",
                slug: "tapas-de-ruedas",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjI4",
                slug: "pernos-de-seguridad",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjMw",
                slug: "llantas-de-aleacion",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjMz",
                slug: "2-tubo-escape",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjM1",
                slug: "antena-tiburon",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjM3",
                slug: "barras-de-techo",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjM5",
                slug: "spoiler-trasero",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjE3",
        slug: "traction",
        inputType: "DROPDOWN",
        valueRequired: false,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTg2",
                slug: "4x2",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTg5",
                slug: "4x4",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTkw",
                slug: "4x2-y-4x4",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjE2",
        slug: "comfort_equipment",
        inputType: "MULTISELECT",
        valueRequired: false,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTc4",
                slug: "sunroof",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTgw",
                slug: "techo-panoramico",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTgx",
                slug: "keyless-boton-de-encendido",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTgz",
                slug: "smart-entry",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTg0",
                slug: "encendido-a-distancia",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTg3",
                slug: "climatizador",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTkx",
                slug: "climatizador-dual",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTkz",
                slug: "climatizador-tri-zona",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTk0",
                slug: "alzavidrios-con-interface",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTk1",
                slug: "espejos-laterales-abatibles-remotamente",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTk2",
                slug: "cargador-inalambrico-de-celular",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTk3",
                slug: "radio-touch",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTk4",
                slug: "entrada-usb-externa-yo-auxiliar",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MTk5",
                slug: "navegador-gps",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjAw",
                slug: "apple-carplay",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjAx",
                slug: "display-de-navegacion",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjAy",
                slug: "volante-regulable",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjAz",
                slug: "volante-calefaccionado",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjA1",
                slug: "comandos-de-audio-al-volante",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjA3",
                slug: "bluetooth-audio-streaming",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjA5",
                slug: "velocidad-crucero",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjEy",
                slug: "limitador-de-velocidad",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjE2",
                slug: "asistente-de-navegacion",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjE4",
                slug: "computador-a-bordo",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjIw",
                slug: "tapiz-de-cuero",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjIy",
                slug: "butacas-electricas",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjIz",
                slug: "butacas-con-memoria",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjI0",
                slug: "asientos-calefaccionados",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjI1",
                slug: "butacas-con-masajeador",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjI3",
                slug: "drive-mode",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjI5",
                slug: "sport-mode",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjMx",
                slug: "grip-control",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjMy",
                slug: "pedales-regulables",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjM0",
                slug: "portalon-electrico",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjM2",
                slug: "trailer-backup",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjM4",
                slug: "volante-cuero",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQw",
                slug: "3-corridas-de-asientos",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQx",
                slug: "alzavidrios-electricos-delanteros",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQy",
                slug: "asiento-electrico",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQz",
                slug: "alzavidrios-manual",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQ3",
                slug: "butacas-de-cuero",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjUx",
                slug: "climatizador-bi-zona",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjUz",
                slug: "control-crucero",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjU1",
                slug: "encendido-boton",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjU4",
                slug: "encendido-llave",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjYw",
                slug: "lcd-delantero-central-con-touchscreen",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjYy",
                slug: "llanta-de-repuesto-full-size",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjY0",
                slug: "retrovisores-exteriores-ajustables-y-retractiles-electricos",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjY2",
                slug: "techo-corredizo-electrico",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjY4",
                slug: "techo-corredizo-manual",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjcw",
                slug: "volante-multifuncional",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjIy",
        slug: "security_equipment",
        inputType: "MULTISELECT",
        valueRequired: false,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQ0",
                slug: "alerta-de-cambio-de-carril",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQ1",
                slug: "alerta-de-impacto-frontal",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQ2",
                slug: "sistema-de-deteccion-de-punto-ciego-bsd",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQ4",
                slug: "reconocimiento-de-senales-de-trafico",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjQ5",
                slug: "velocidad-crucero-adaptativa-eyesigth",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjUw",
                slug: "freno-de-mano-electronico",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjUy",
                slug: "hill-assist",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjU0",
                slug: "park-assist",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjU2",
                slug: "frenos-de-disco-en-las-4-ruedas",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjU3",
                slug: "abs",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjU5",
                slug: "repartidor-electronico-de-frenado",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjYx",
                slug: "asistencia-al-frenado-de-urgencia-afu-ebs-hba-bas",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjYz",
                slug: "control-de-traccion-asr",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjY1",
                slug: "control-de-estabilidad-esp",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjY3",
                slug: "anclaje-isofix-para-ninos",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MjY5",
                slug: "alzadores-para-ninos-incorporado",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjcx",
                slug: "sensores-de-retroceso",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjcy",
                slug: "sensores-de-proximidad-frontal",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjcz",
                slug: "camara-retroceso",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjc0",
                slug: "camara-de-retroceso-en-180o",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjc1",
                slug: "camara-de-retroceso-en-360o",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjc2",
                slug: "alarma-antirrobo",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjc3",
                slug: "sistema-inmovilizador-antirrobo",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjc4",
                slug: "sensor-de-lluvia",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjc5",
                slug: "espejo-retrovisor-fotosensible",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjgw",
                slug: "sensor-de-presion-de-neumaticos",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjgx",
                slug: "neblineros",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjgy",
                slug: "airbag-conductor",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjgz",
                slug: "airbag-pasajero",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjg0",
                slug: "asistencia-en-pendientes",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjg1",
                slug: "airbags-laterales-y-airbags-de-cortina-superior",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjg2",
                slug: "barras-protectoras-puertas-laterales",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjg3",
                slug: "cinturon-3-puntos",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjg4",
                slug: "cierre-automatico-de-puertas-en-velocidad",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjg5",
                slug: "carroceria-con-zonas-de-deformacion-programada",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjkw",
                slug: "encendido-automatico-de-luces",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjkx",
                slug: "frenado-automatico-de-emergencia",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjky",
                slug: "limpia-parabrisas-trasero",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjkz",
                slug: "sensores-de-proximidad-delanteros-yo-traseros",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjIz",
        slug: "doors",
        inputType: "DROPDOWN",
        valueRequired: false,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjk0",
                slug: "2",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjk1",
                slug: "3",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjk2",
                slug: "4",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjk3",
                slug: "5",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjk4",
                slug: "6",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6Mjk5",
                slug: "0",
              },
            },
          ],
        },
      },
    },
    {
      attribute: {
        id: "QXR0cmlidXRlOjI0",
        slug: "basic_equipment",
        inputType: "MULTISELECT",
        valueRequired: false,
        choices: {
          edges: [
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MzAw",
                slug: "llaves",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MzAx",
                slug: "volante",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MzAy",
                slug: "radio",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MzAz",
                slug: "aire-acondicionado",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MzA0",
                slug: "cierre-centralizado",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MzA1",
                slug: "alzavidrios-electricos-total",
              },
            },
            {
              node: {
                id: "QXR0cmlidXRlVmFsdWU6MzA2",
                slug: "espejos-laterales-electricos",
              },
            },
          ],
        },
      },
    },
  ],
  taxClass: {
    id: "VGF4Q2xhc3M6MQ==",
  },
}
```

### What did you expect to happen?

I expect to import Products and their Variants including all their attributes values and choices without problems.

It's important to notice the bug was introduced somewhere between the versions (`3.19.4`, `3.19.10`]

### Logs

![imagen](https://github.com/saleor/saleor/assets/897389/35ac6f5a-7884-4a7e-ac96-dfbb48f54138)

> Error Report
```js
{
    "error": {
        "name": "CombinedError",
        "graphQLErrors": [
            {
                "message": "Some values are not from the provided attribute.",
                "locations": [
                    {
                        "line": 2,
                        "column": 3
                    }
                ],
                "path": [
                    "created"
                ],
                "extensions": {
                    "exception": {
                        "code": "GraphQLError",
                        "stacktrace": [
                            "graphql.error.base.GraphQLError: Some values are not from the provided attribute."
                        ]
                    }
                }
            }
        ],
        "response": {}
    },
    "input": {
        "sku": "2596",
        "product": "UHJvZHVjdDoxNQ==",
        "attributes": [
            {
                "id": "QXR0cmlidXRlOjM3",
                "values": [
                    "diesel"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjM5",
                "values": [
                    "suv"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjQw",
                "values": [
                    "6"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjQy",
                "plainText": "2200"
            },
            {
                "id": "QXR0cmlidXRlOjM4",
                "values": [
                    "automatica"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjQ2",
                "values": [
                    "4"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjQ3",
                "values": [
                    "aire-acondicionado",
                    "cierre-centralizado",
                    "alzavidrios-electricos-total"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjQz",
                "plainText": "188"
            },
            {
                "id": "QXR0cmlidXRlOjQx",
                "values": [
                    "4x4"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjQ0",
                "values": [
                    "llantas-de-aleacion",
                    "spoiler-trasero"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjQ1",
                "values": [
                    "keyless-boton-de-encendido",
                    "climatizador",
                    "entrada-usb-externa-yo-auxiliar",
                    "navegador-gps",
                    "volante-regulable",
                    "volante-calefaccionado",
                    "asientos-calefaccionados",
                    "volante-cuero",
                    "alzavidrios-electricos-delanteros",
                    "asiento-electrico",
                    "butacas-de-cuero",
                    "climatizador-bi-zona",
                    "control-crucero",
                    "encendido-boton",
                    "lcd-delantero-central-con-touchscreen",
                    "retrovisores-exteriores-ajustables-y-retractiles-electricos",
                    "techo-corredizo-electrico",
                    "volante-multifuncional"
                ]
            },
            {
                "id": "QXR0cmlidXRlOjQ4",
                "values": [
                    "sistema-de-deteccion-de-punto-ciego-bsd",
                    "abs",
                    "asistencia-al-frenado-de-urgencia-afu-ebs-hba-bas",
                    "control-de-traccion-asr",
                    "control-de-estabilidad-esp",
                    "anclaje-isofix-para-ninos",
                    "camara-retroceso",
                    "sistema-inmovilizador-antirrobo",
                    "sensor-de-lluvia",
                    "neblineros",
                    "airbag-pasajero",
                    "asistencia-en-pendientes",
                    "airbags-laterales-y-airbags-de-cortina-superior",
                    "barras-protectoras-puertas-laterales",
                    "cierre-automatico-de-puertas-en-velocidad",
                    "carroceria-con-zonas-de-deformacion-programada",
                    "encendido-automatico-de-luces",
                    "frenado-automatico-de-emergencia",
                    "limpia-parabrisas-trasero",
                    "sensores-de-proximidad-delanteros-yo-traseros"
                ]
            }
        ],
        "quantityLimitPerCustomer": 1,
        "externalReference": "2596",
        "trackInventory": true,
        "preorder": null,
        "stocks": [
            {
                "warehouse": "V2FyZWhvdXNlOmIxODRlZTM2LTI1YjQtNGU0NS1iNmQxLTg3OWY3OWZhMjk5MA==",
                "quantity": 1
            }
        ],
    }
}
```

### Environment

Saleor version: 3.19.10
OS and version: Docker (OS X)

**Comment by aakashkejriwal** [comment]
👍  I'm getting a bunch of issues with bulk uploads since recent upgrade. I'm currently on v3.19.11. Sample GQL I'm running:

```
mutation ProductUpdate($id: ID!, $input: ProductInput!) {
  productUpdate(id: $id, input: $input) {
    errors {
      ...ProductErrorWithAttributes
      __typename
    }
    product{
      id
      name
      product_parent_type: attribute(slug:"product_parent_type"){
        attribute{
          choices(first:5){
            edges{
              node{
                id
                name
              }
            }
          }
        }
        values{
          id
          externalReference
          name
        }
      }
      product_sibling_type: attribute(slug:"product_sibling_type"){
        values{
          id
          externalReference
          name
        }
      }
      product_sibling_group_id: attribute(slug:"product_sibling_group_id"){
        attribute{
          id
        }
        values{
          id
          externalReference
          name
        }
      }

    }
    __typename
  }
}

fragment ProductErrorWithAttributes on ProductError {
  ...ProductError
  attributes
  __typename
}

fragment ProductError on ProductError {
  code
  field
  message
  __typename
}

```




I also noticed another inconsistency:

Setting multiselect with multiple choices using external references doesn't seem to be working. It only sets one.

Not sure if this also cropped up during the upgrade.

I think it maybe because of some changes in this  as its the only one with release notes related to attributes - https://github.com/saleor/saleor/pull/15493

[IKarbowiak](https://github.com/IKarbowiak) - would you have any idea about this?

**Comment by yonathan9669** [comment]
Update: This bug was resolved on version `3.19.13`
